components/SegmentedControl/src/Segmented/Segmented.Properties.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.System;
+
+namespace CommunityToolkit.Labs.WinUI;
+
+public partial class Segmented : ListViewBase
+{
+    /// <summary>
+    /// The backing <see cref="DependencyProperty"/> for the <see cref="AutoSelection"/> property.
+    /// </summary>
+    public static readonly DependencyProperty AutoSelectionProperty = DependencyProperty.Register(
+        nameof(AutoSelection),
+        typeof(bool),
+        typeof(Segmented),
+        new PropertyMetadata(defaultValue: true));
+
+    /// <summary>
+    /// Gets or sets the icon.
+    /// </summary>
+    public bool AutoSelection
```

Comment:
@michael-hawker Do you have a better name for this :)?

Reply:
Do we need this property? Since we're inheriting from `ListViewBase` we already have `SelectedItem`/`SelectedIndex`, so if they're not set we don't select anything by default. (i.e. remove the logic around L102 in the segemented.cs)

Then we just fix the issue with those not working for whatever reason instead? Chances can be that someone's going to use those anyway to rehydrate a state so then it'd conflict with having this sort of property as well?

Thoughts?